vulcan/_message.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from related import (
+    IntegerField,
+    StringField,
+    DateField,
+    ChildField,
+    immutable,
+    to_model,
+)
+
+from ._subject import Subject
+from ._teacher import Teacher
+from ._utils import sort_and_filter_date
+@immutable
+class Message:
+	sender = StringField(key="Nadawca")
+	senderid = StringField(key="NadawcaId")
+	addressees = StringField(key="Adresaci")
+	title = StringField(key="Tytul")
+	content = StringField(key="Tresc")
+	postdate = StringField(key="DataWyslania")
+	posthour = StringField(key="GodzinaWyslania")
+	folder= StringField(key="FolderWiadomosci")
```

Comment:
Use snake_case for variable and method names, like `post_date` instead of `postdate`.
And I have a proposal for other names for these attributes:
`recipients` instead of `addressees`
`sent_date` instead of `postdate` (use here `DateField`, not `StringField`)
`sent_time` instead of `posthour` (use here `TimeField`, not `StringField`)
`sender_id` instead of `senderid` (use here `IntegerField`)
Move `sender` to the bottom and make it `ChildField` for `Teacher` with `required=False`